packages/admin/admin/src/form/FinalFormSelect.tsx

```diff
@@ -102,9 +105,9 @@ export const FinalFormSelect = <T,>({
             value={Array.isArray(value) ? value.map((i) => getOptionValue(i)) : getOptionValue(value)}
         >
             {loading && (
-                <MenuItem value="" disabled>
+                <MenuItemDisabledOverrideOpacity value="" disabled>
```

Comment:
Can't we remove the disabled instead? If not, I'd use `sx` here.

Reply:
If we remove the disabled one can click the loading MenuItem. I think we don't want this behaviour

https://github.com/user-attachments/assets/d78e5d3e-ff65-47a4-b583-ecdd632a798d